.github/workflows/perf_analysis.yml

```diff
@@ -0,0 +1,83 @@
+name: perf_analysis
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release/**'
+  schedule:
+    # Once a day at 02:00, should be later than `perf_micro.yml`.
+    - cron: '0 2 * * *'
```

Comment:
I'm not sure that we really need it once a day (instead of byweakly, for example). But I'm not insist.

Reply:
I propose to run analytics once per day and analyze it as soon someone have a spare time for this activity. In such case, we will have fresh data for human analysis. Anyway, this workflow is cheap from machine resources point of view.